platform/viewer/cypress/integration/measurement-tracking/OHIFStudyBrowser.spec.js

```diff
@@ -52,7 +52,7 @@ describe('OHIF Study Viewer Page', function() {
 
   it('performs double-click to load thumbnail in active viewport', () => {
     // Have to finish rendering the image before this works
-    cy.wait(250);
+    cy.wait(350);
```

Comment:
This test is not actually checking anything right?

Reply:
It performs a double click - but the double click has to wait for the viewer to be ready to display something.